python/cugraph/cugraph/structure/graph_implementation/simpleDistributedGraph.py

```diff
@@ -89,51 +88,58 @@ def _make_plc_graph(
         src_col_name,
         dst_col_name,
         store_transposed,
-        num_edges,
     ):
 
         weights = None
         edge_ids = None
         edge_types = None
+        num_arrays = len(edata_x)
 
         if simpleDistributedGraphImpl.edgeWeightCol in edata_x[0]:
-            weights = _get_column_from_ls_dfs(
-                edata_x, simpleDistributedGraphImpl.edgeWeightCol
```

Comment:
Can you also remove the now dead function `_get_column_from_ls_dfs ` please.

Reply:
Right,  I removed that locally. I will push my changes shortly